examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+"""
+Mask and Plot Remote Sensing Data with EarthPy
+==============================================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
+# ---------------------
+#
+# .. note:: Below we walk through a typical workflow using Landsat data with EarthPy.
+
+
+###############################################################################
+# Import Packages
+# ------------------------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial, plot and mask.
+
+from glob import glob
+import os
+import matplotlib.pyplot as plt
+import rasterio as rio
+from rasterio.plot import plotting_extent
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+import earthpy.mask as em
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+###############################################################################
+# Import Example Data
+# ------------------------------
+# To get started, make sure your directory is set. Create a stack from all of the
+# Landsat .tif files (one per band) and import the ``landsat_qa`` layer which provides
+# the locations of cloudy and shadowed pixels in the scene.
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the landsat bands
```

Comment:
May need to have the 79 `#` signs above this to prevent it from being rendered as a comment in a python code chunk.

Reply:
the intent here was all of the data import and processing in one section. thus I did not intend to create a section here.